spec/diagnose_spec.rb

```diff
@@ -690,25 +766,16 @@
   end
 
   it "submitted report contains agent diagnostics errors" do
-    matchers =
-      case @runner.type
-      when :nodejs
-        {
-          "extension" => {
-            "config" => {
-              "valid" => {
-                "error" => "RequiredEnvVarNotPresent(\"_APPSIGNAL_PUSH_API_KEY\")",
-                "result" => false
-              }
-            }
+    expect_report_for(
+      :agent,
+      "extension" => {
+        "config" => {
+          "valid" => {
+            "error" => "RequiredEnvVarNotPresent(\"_APPSIGNAL_PUSH_API_KEY\")",
+            "result" => false
           }
         }
-      when :ruby, :elixir
-        # TODO
-        raise "Report matchers missing"
-      else
-        raise "No clause for runner #{@runner}"
-      end
-    expect_report_for(:agent, matchers)
+      }
+    )
```

Comment:
No. I think the test will always return the same report because it's the output from the extension. I'll run into it again when updating this for the Elixir integration if it requires a different assertion.